kurl_proxy/cmd/main.go

```diff
@@ -53,8 +52,6 @@ type cert struct {
 func main() {
 	log.Printf("Commit %s\n", os.Getenv("COMMIT"))
 
-	rand.Seed(time.Now().UnixNano())
```

Comment:
Why remove this?

Reply:
> Why remove this?

Accidental clean up as I was removing deprecated code. I meant to put it back.

We however do not need it. It was deprecated in go1.20. The runtime automatically generates a seed for us. Also, I do not see anywhere where we use random numbers.

Reply:
I think we are safe to remove this unless there are objections.

I reviewed the code and do not see any reason for keeping it